src/components/SearchBar/SearchBar.stories.mdx

```diff
@@ -0,0 +1,293 @@
+import HelperErrorText from "../HelperErrorText/HelperErrorText";
+import Button from "../Button/Button";
+import { ButtonTypes } from "../Button/ButtonTypes";
+import Icon from "../Icons/Icon";
+import { IconNames } from "../Icons/IconTypes";
+import Input from "../Input/Input";
+import { InputTypes } from "../Input/InputTypes";
+import * as stories from "../Autosuggest/Autosuggest.stories.tsx";
+
+<Meta
+  title="SearchBar"
+  component={SearchBar}
+  decorators={[withDesign]}
+  parameters={{
+    design: {
+      type: "figma",
+      url:
+        "https://www.figma.com/file/qShodlfNCJHb8n03IFyApM/Master?node-id=11689%3A423",
+    },
+  }}
+  argTypes={{
+    blockName: {
+      table: { disable: true },
+    },
+  }}
+/>
+
+# SearchBar
+
+<Description of={SearchBar} />
+
+The main wrapper element that is rendered is `<form>` DOM element. All the props
+passed to `SearchBar` will apply to the `<form>` element. If you need to update
+the children, make sure the props are passed to the appropriate components.
+
+The `Select` dropdown narrows the search within a specific category, typically
+title or author. You should only have one `Select` component at a time. If you
+need more options, link to the advance search page. Toggle the `Select` through
+the Controls.
+
+_Note: The `HelperErrorText` component should be the next direct sibling of the
+`SearchBar` component if it will be used._
+
+export const optionsGroup = {
+  Art: [
+    "Academic Painting",
+    "Amazing Painting",
+    "Ancient Statue",
+    "Basic Painting",
+    "Beautiful Statue",
+  ],
+  Bushes: [
+    "Blue-Hydrangea Bush",
+    "Blue-Hydrangea Start",
+    "Holly Bush",
+    "Orange-Tea-Olive Bush",
+    "Pink Camellia Bush",
+  ],
+  Clothing: ["3D Glasses", "Academy Uniform", "Acid-Washed Jacket"],
+  "DIY Recipes": ["Acorn Pochette", "Acoustic Guitar", "Angled Signpost"],
+  Flowers: [
+    "Black Cosmos",
+    "Blue Pansies",
+    "Gold Roses",
+    "Lily of the Valley",
+    "Orange Hyacinth",
+  ],
+  Fossils: [
+    "Acanthostega",
+    "Amber",
+    "Ammonite",
+    "Ankylo Skull",
+    "Anomalocaris",
+    "Australopith",
+  ],
+  Fruits: ["Apples", "Oranges", "Pears", "Coconuts", "Cherries"],
+  Furniture: [
+    "Abstract Wall",
+    "Accessories Stand",
+    "ACNH Switch",
+    "Air Circulator",
+  ],
+  Materials: [
+    "Acorn",
+    "Aquaris Fragment",
+    "Bamboo Piece",
+    "Boot",
+    "Clump of Weeds",
+  ],
+  NPC: ["Celeste", "Kicks", "Leif", "Redd", "Saharah"],
+  Songs: ["Agent K.K.", "Café K.K.", "Comrade K.K."],
+  Tools: [
+    "Axe",
+    "Bamboo Wand",
+    "Fishing Rod",
+    "Shovel",
+    "Slingshot",
+    "Watering Can",
+  ],
+  Villagers: ["Agnes", "Alfonso", "Anchovy", "Axel", "Aurora"],
+};
+
+<Preview withToolbar>
+  <Story
+    name="Basic"
+    args={{
+      showHelperText: true,
+      showSelect: true,
+      isErrorState: false,
+      disabledButton: false,
+    }}
+  >
+    {args => (
+      <>
+        <SearchBar {...args}>
```

Comment:
As I was mentioning in the Slack channel, we might want to switch to DIV tags instead of fragment tags only until Storybook stops rendering fragments in the source code areas.  The idea is that the code there should be entire copy/paste-able.